transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
+    """
+    TYPE_A = 'A'
+    TYPE_AAAA = 'AAAA'
+    TYPE_CNAME = 'CNAME'
+    TYPE_MX = 'MX'
+    TYPE_NS = 'NS'
+    TYPE_TXT = 'TXT'
+    TYPE_SRV = 'SRV'
```

Comment:
Yeah, I think these might be the ones defined by the core RFC, there are a lot of addenda to the core RFC for DNS. Lets leave as-is.